linux_os/guide/services/ssh/ssh_server/sshd_allow_only_protocol2/oval/shared.xml

```diff
@@ -0,0 +1,45 @@
+<def-group>
+  <definition class="compliance" id="sshd_allow_only_protocol2" version="1">
+    <metadata>
+      <title>Ensure Only Protocol 2 Connections Allowed</title>
+      <affected family="unix">
+        <platform>multi_platform_wrlinux</platform>
+        <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_rhv</platform>
+        <platform>multi_platform_debian</platform>
+        <platform>multi_platform_ubuntu</platform>
+        <platform>multi_platform_ol</platform>
+      </affected>
+      <description>The OpenSSH daemon should be running protocol 2.</description>
+    </metadata>
+    <criteria comment="SSH is configured correctly or is not installed"
+    operator="OR">
+      <criteria comment="sshd is not installed" operator="AND">
```

Comment:
We could use Jinja macros application_not_required_or_requirement_unset and application_required_or_requirement_unset here. The latter doesn't even contain the closing criteria tag, so we could easily sneak in the extend_definition element for the version check.

But I don't know if it would be useful for anything.

Reply:
I would not use this macro because it has hardcoded descriptions connected with sshd and it would only confuse people. I don't think it brings any benefit. But I will add macro affected_oval.

Reply:
OK, makes sense